packages/middleware/README.md

```diff
@@ -0,0 +1,30 @@
+<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/middleware
+
+**A core Josh utility to create middleware**
+
+[![GitHub](https://img.shields.io/github/license/josh-development/utilities)](https://github.com/josh-development/utilities/blob/main/LICENSE)
+[![npm](https://img.shields.io/npm/v/@joshdb/middleware?color=crimson&logo=npm&style=flat-square&label=@joshdb/middleware)](https://www.npmjs.com/package/@joshdb/middleware)
+
+</div>
+
+## Description
+
+A core Josh utility to create middleware.
+
+## Features
+
+- Written in TypeScript
+- Offers CommonJS, ESM, and UMD bundles
+- Fully tested
+
+## Installation
+
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
+
+```sh
+npm install @joshdb/middleware
+```
```

Comment:
Add yarn to keep consistency across readmes

Reply:
It is consistent?

Reply:
I see it in @josh-development/core

Reply:
That's to be done